templates/layout/parts/page_header.html.twig

```diff
@@ -36,6 +36,11 @@
 {% set is_vertical = user_pref('page_layout') == 'vertical' %}
 
 <body class="{{ user_pref('fold_menu') and is_vertical ? 'navbar-collapsed' : '' }} {{ is_vertical ? 'vertical-layout' : 'horizontal-layout' }} {{ is_debug_active ? 'debug-active' : '' }}">
+   {% if constant('BYPASSING_UPDATES') %}
```

Comment:
Maybe using `{% if constant('SKIP_UPDATES') and call('Update::isDbUpToDate') %}` could prevent usage a a new global constant.